package.json

```diff
@@ -1,24 +1,29 @@
 {
-  "name": "airnode",
+  "name": "api3",
```

Comment:
Are you sure this simple renaming works?

Reply:
Haven't tested all the packages extensively but the deployer (which is using 2-3 others) was working fine.